app/views/edit/deployment-config.html

```diff
@@ -0,0 +1,366 @@
+                                  <lifecycle-hook model='strategyData[strategyParamsPropertyName].post'
+                                                    type='post'
+                                                    available-volumes='volumeNames'
+                                                    available-containers='containerNames'
+                                                    namespace="projectName">
+                                  </lifecycle-hook>
+                                </div>
+                              </div>
+                            </div>
+                          </div>
+                          <div class="gutter-top">
+                            <div ng-if="!view.advancedStrategyOptions">To set additional parameters or edit lifecycle hooks, view <a href="" ng-click="view.advancedStrategyOptions = true">advanced strategy options.</a></div>
+                            <a ng-if="view.advancedStrategyOptions" href="" ng-click="view.advancedStrategyOptions = false">Hide strategy advanced options</a>
+                          </div>
+                        </dl>
+                      </div>
+
+
+                      <div class="section">
+                        <h3 class="with-divider">Images</h3>
+                        <dl class="dl-horizontal left">
+
+                          <div ng-repeat="(containerName, containerConfig) in containerConfigByName">
+
+                            <div class="container-name">
+                              <h4>Container {{containerName}}</h4>
+                            </div>
+                            <div class="form-group" ng-if="containerConfig.hasDeploymentTrigger">
+                              <label class="required">Image Change Trigger</label>
+                              <istag-select model="containerConfig.triggerData.istag" select-disabled="disableInputs" include-shared-namespace="true"></istag-select>
+                              <span class="help-block">New deployments will automatically start when the image stream tag is updated.</span>
+                            </div>
+
+                            <div ng-if="!containerConfig.hasDeploymentTrigger">
+                              <label for="imageName" class="required">Image Name</label>
+                              <div>
+                                <input class="form-control"
+                                  id="imageName"
+                                  name="imageName"
+                                  ng-model="containerConfig.image"
+                                  type="text"
+                                  autocorrect="off"
+                                  autocapitalize="off"
+                                  spellcheck="false"
+                                  required>
+                              </div>
+                              <span class="help-block">New deployment can be triggered only manually.</span>
```

Comment:
This isn't really true since there could be config change triggers as well. I'd just remove the help here.